src/modules/base.js

```diff
@@ -0,0 +1,77 @@
+import { createTab } from './helper';
+
+const base = () => {
+  const content = document.querySelector('div#content');
+  const section = document.createElement('section');
+
+  function nav() {
+    const heroHead = document.createElement('div');
+    heroHead.setAttribute('class', 'hero-head');
+    section.appendChild(heroHead);
+
+    const navbar = document.createElement('nav');
+    navbar.setAttribute('class', 'navbar');
+    heroHead.appendChild(navbar);
+
+    const container = document.createElement('div');
+    container.setAttribute('class', 'container');
+    navbar.appendChild(container);
+
+    const navbarBrand = document.createElement('div');
+    navbarBrand.setAttribute('class', 'navbar-brand');
+    container.appendChild(navbarBrand);
+
+    const navbarItem = document.createElement('a');
+    navbarItem.setAttribute('class', 'navbar-item');
+    navbarBrand.appendChild(navbarItem);
+
+    const logoImg = document.createElement('img');
+    logoImg.setAttribute('src', './assets/images/ihop.png');
+    logoImg.setAttribute('alt', 'ihop logo');
+    navbarItem.appendChild(logoImg);
+
+    const menuHero = document.createElement('div');
+    menuHero.setAttribute('class', 'navbar-menu');
+    menuHero.setAttribute('id', 'navbarMenuHeroB');
+    container.appendChild(menuHero);
+
+    const navbarEnd = document.createElement('div');
+    navbarEnd.setAttribute('class', 'navbar-end');
+    navbarEnd.setAttribute('id', 'here-tab');
+    menuHero.appendChild(navbarEnd);
+  }
+
+  function addSection() {
+    section.setAttribute('class', 'hero is-info is-large');
+    content.appendChild(section);
+  }
```

Comment:
Use arrow functions instead of the `function` keyword. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))